networking/external_dns_operator/nw-control-dns-records-public-hosted-zone-infoblox.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-creating-dns-records-on-infoblox.adoc
+
+:_content-type: PROCEDURE
+[id="nw-control-dns-records-public-dns-zone-infoblox_{context}"]
+= Creating DNS records on an public DNS zone for Infoblox by using Red Hat External DNS Operator
+
+You can create DNS records on a public DNS zone for Infoblox by using the Red Hat External DNS Operator.
+
+.Procedure
+
+. Create a `secret` with Infoblox credentials:
++
+----
+$ oc -n external-dns-operator create secret generic infoblox-credentials --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_USERNAME=${INFOBLOX_USERNAME} --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_PASSWORD=${INFOBLOX_PASSWORD}
+----
+
+. Get the routes to check your cluster domain:
++
+[source,terminal]
+----
+$ oc get routes --all-namespaces | grep console
+----
++
+.Example Output
+----
+openshift-console          console             console-openshift-console.apps.aws.devcluster.openshift.com                       console             https   reencrypt/Redirect     None
+openshift-console          downloads           downloads-openshift-console.apps.aws.devcluster.openshift.com                     downloads           http    edge/Redirect          None
+----
+
+.From the UI
+
+. Add an authoritative DNS zone for your cluster domain to Infoblox using the Infoblox web console.
+
+.. Click *Data Management* -> *DNS*-> *Add* -> *Zone* -> *Authoritative Zone* -> *Forward Mapping*.
+.. Enter the cluster domain as zone name (e.g. `aws.devcluster.openshift.com`).
+.. Click *+*.
+.. Click *Save*.
+.. Click *Close*.
```

Comment:
This can be removed, we assume the zone is there, explaining how to create a zone in Infoblox is out of the scope of OCP docs.
The zone is filtered by its domain in the next step, so it's visible enough.

Reply:
Ack'ed- Removed the `From the UI` part